### DISCUSSION TITLE
Organisation when bootstrapping

### DISCUSSION THREAD

**Discussion by musicformellons** [zitadel/zitadel#8297] [Q&A]
Following is just to make sure I'm not missing something.

Within zitadel-terraform I found issues with examples how to bootstrap 'as much as currently possible' a Zitadel instance with resources. It is done in two steps: 1. initialization of zitadel 2. create (additional) resources with terraform. [Requirements](https://zitadel.com/docs/guides/manage/terraform-provider#before-you-start) before anything can be done with terraform are: 1. zitadel instance 2. service user. So these are created by using (docker-compose or config files) during the 'initialization of zitadel'. 

AFAICT creating that service user means there is also an organisation created (by default named ZITADEL). The only thing that can be retrieved from this to be used within terraform is the exported machinekey (which contains a userid). So this means that when I want to bootstrap an organisation I can not use the already created ZITADEL organisation.

So am I correct in this: when following the current bootstrapping procedure you will always have this firstinstance 'bootstrap' organisation which you can not proceed with 'lying around', so basically you need to use terraform to setup another organisation you can create resources for?

**Comment by hifabienne** [comment]
the first organization is used for zitadel internal as well, in there is a project called zitadel with some apps, which make it possible to use the management console on the backends of zitadel.
I am not sure if I understand what your question is, but we recommend setting up other organizations for your business logic
